src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -84,6 +88,49 @@ void testInternalUserScenario() {
         updateOrgStatusScenarios();
     }
 
+    @Test
+    @DisplayName("PRD Internal Test for Group Access Scenarios")
+    void testGroupAccessInternalScenario() {
+        setUpTestData();
+        createOrganisationScenario();
+        inviteMultipleUserScenarios();
+        findUserInternalScenarios();
+    }
+
+    public void inviteMultipleUserScenarios() {
+        inviteUserByAnInternalOrgUser(generateRandomEmail());
+        inviteUserByAnInternalOrgUser(generateRandomEmail());
+        inviteUserByAnInternalOrgUser(generateRandomEmail());
+        inviteUserByAnInternalOrgUser(generateRandomEmail());
+    }
+
+    public void findUserInternalScenarios() {
+        findByUserIdOrAndSinceDate(null, invitedUserIds.get(0));
+        findByUserIdOrAndSinceDate(lastUpdateTime, null);
+        findByUserIdOrAndSinceDate(lastUpdateTime, invitedUserId);
+        findByUserIdOrAndSinceDate(null, null);
+
+        findBySinceDatePageSizeOrAndSearchAfter(lastUpdateTimes.get(0), "3", null);
+        findBySinceDatePageSizeOrAndSearchAfter(lastUpdateTimes.get(0), "1", lastRecordIdInPage);
+        findBySinceDatePageSizeOrAndSearchAfter(lastUpdateTimes.get(0), null, lastRecordIdInPage);
+
+        findByUserIdInvalidS2SToken(invitedUserIds.get(0));
+        findByUserIdNotFound("non-existing-id");
+    }
+
+    public NewUserCreationRequest inviteUserByAnInternalOrgUser(String email) {
```

Comment:
method return type not used in caller , make it void.

Reply:
fixed.